### DISCUSSION TITLE
Main 페이지 관련

### DISCUSSION THREAD

**Discussion by silverpoodle** [goalapa/spring-cacamuca#74] [General]
메인화면에서
<img width="500" alt="스크린샷 2023-07-19 081704" src="https://github.com/goalapa/spring-cacamuca/assets/88484476/940e68f4-d960-4a29-b867-dc0150e91e86">
이렇게 만드려고 하는데
이러면 식품이나 리뷰 db에 접근할 수 있어야 하니까
mainDomain을 따로 만들어야하는건가요???🤔

**Comment by myrhymetree** [comment]
식품이나 리뷰 도메인을 이용하거나 식품이나 리뷰에서 만든 api를 이용해야 할 것 같습니다

**Comment by bbbbooo** [comment]
이미지는 reviewPic에 있어서 아마 리뷰쪽에서 접근해야될거 같습니다
